app/views/chargeback_rate/_cb_rate_show.html.haml

```diff
@@ -49,9 +49,9 @@
         %td{:rowspan => num_tiers}
           = h(rate_detail_group(detail.chargeable_field[:group]))
         %td{:rowspan => num_tiers}
-          = detail.chargeable_field[:description]
+          = _(detail.chargeable_field[:description])
```

Comment:
Where are the strings that show here being collected for translation?

Reply:
I guess these strings should also be added to `en.yml` as well?

Reply:
Similar thing as above -- we need to know where the strings that we want to see translated here come from. A yaml file perhaps?

Reply:
Content of `@record.rate_type` on line 16 and `detail.chargeable_field[:description]` come from `db/fixtures/chargeable_fields.yml` and `db/fixtures/chargeback_rates.yml` (both found in core).

Have these 2 files been configured for string extraction?

Reply:
> Have these 2 files been configured for string extraction?

By that do you mean adding those files to `locale_task_config.yaml`?

Reply:
@DavidResende0 yes

Reply:
done